src/main/java/rcms/utilities/daqaggregator/mappers/ObjectMapper.java

```diff
@@ -418,8 +424,11 @@ public String toString() {
 
 		Set<String> frlPcs = new HashSet<String>();
 
-		Map<Long, rcms.utilities.hwcfg.eq.FRL> frls = daqPartition.getDAQPartitionSet().getEquipmentSet().getFRLs();
-		for (rcms.utilities.hwcfg.eq.FRL hwfrl : frls.values()) {
+		/*Map<Long, rcms.utilities.hwcfg.eq.FRL> frls = daqPartition.getDAQPartitionSet().getEquipmentSet().getFRLs();
```

Comment:
Oops, sorry. It was left after the test to compare outputs. I'll remove these occurences for the next merge.